pyproject.toml

```diff
@@ -191,16 +188,17 @@ version = "^3.1"
 optional = true
 
 [tool.poetry.dependencies.tensorflow-text]
-version = "~2.6"
+version = "^2.7.0"
 markers = "sys_platform!='win32'"
 
 [tool.poetry.dependencies."github3.py"]
 version = "~1.3.0"
 optional = true
 
 [tool.poetry.dependencies.transformers]
-version = ">=2.4,<2.12"
+version = "~4.13.0"
 optional = true
+extras = [ "sentencepiece",]
```

Comment:
Does this mean that if a user did `pip install rasa[transformers]`, it wouldn't bring in `sentencepiece` automatically?

Reply:
No, it means that it will bring in `transformers`, and we've specified transformers should be brought in with it's `sentencepiece` extra - so basically what we've done is outsourced installing `sentencepiece` to `transformers` since it isn't an explicit dependency for us but one we have because of `transformers` in the first place. Doesn't make sense for us to play catch up and mirror their dependency stuff into our project every time we need to update `transformers`

Reply:
Makes perfect sense, thanks for clarifying 🙏